src/api/handler.rs

```diff
@@ -0,0 +1,8 @@
+use serde_json::json;
+use warp::reply::json;
+use warp::{Rejection, Reply};
+
+// health handler, responds with { ok: true }
```

Comment:
Doc comments are `///` :)

Reply:
I didn't intend for this to be a doc comment, but since it's public, it probably should be.

Reply:
actually scratch that, it's only public to the `api` own mod. Should we be doc-commenting all functions, public and otherwise?

Reply:
You're right, I'm sorry. :) I was being overly pedantic.